manifest.webapp

```diff
@@ -65,6 +65,16 @@
       "description": "Used in services to start other services",
       "type": "io.cozy.jobs",
       "verbs": ["POST"]
+    },
+    "dacc": {
+      "type": "cc.cozycloud.dacc",
+      "verbs": ["POST"],
+      "description": "Remote-doctype required to send anonymized CO2 contributions"
+    },
+    "dacc-dev": {
```

Comment:
est-ce qu'il y a besoin de documenter quelque chose ?

Reply:
Done in [bc1eec7](https://github.com/cozy/coachCO2/pull/134/commits/bc1eec741fd8587bcec89898bd8a6ac8695d0f93)